vault/activity_log_test.go

```diff
@@ -4679,3 +4681,49 @@ func TestActivityLog_writePrecomputedQuery(t *testing.T) {
 	require.Equal(t, 1, monthRecord.NewClients.Counts.EntityClients)
 	require.Equal(t, 1, monthRecord.NewClients.Counts.NonEntityClients)
 }
+
+type mockTimeNowClock struct {
```

Comment:
Does this need to implement all the `Clock` functions?

Reply:
since it embeds `timeutil.DefaultClock`, any functions which aren't explicitly defined will use the DefaultClock's implementation

Reply:
Ah, right! I did not closely looked into the struct. Thanks for the clarification!